src/rgw/setup/rgw_service.py

```diff
@@ -40,4 +41,4 @@ def start(conf: MappedConf, config_file, log_file, motr_trace_dir, rgw_cwd, inde
             raise SetupError(e.errno, "Failed to start radosgw service. %s", e)
         except Exception as e:
             Log.error(f"Failed to start radosgw service:{e}")
-            raise SetupError(e, "Failed to start radosgw service. %s", e)
\ No newline at end of file
+            raise SetupError(e, "Failed to start radosgw service. %s", e)
```

Comment:
empty line was added here. 
eariler it was not there hence it was showing red mark . hence fixed in this patch